easybuild/easyconfigs/f/FIX/FIX-1.068-foss-2019a-Python-3.7.2.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'Tarball'
+
+name = 'FIX'
+version = '1.068'
```

Comment:
@crubb the latest version is 1.06.12 (that is what I got when I download it)...

Reply:
Fixed.